sauce/features/budget/enlarge-categories-dropdown/index.js

```diff
@@ -0,0 +1,53 @@
+import { Feature } from 'toolkit/core/feature';
+import * as toolkitHelper from 'toolkit/helpers/toolkit';
+
+const DEFAULTADDITIONALHEIGHT = 100; // 4 pixels of padding
```

Comment:
pretty nitpicky but if we could use YELLING_SNAKE_CASE instead of just YELLINGCASE That would be good :D